src/main/java/com/swervedrivespecialties/exampleswerve/util/BeakXboxController.java

```diff
@@ -1,13 +1,50 @@
 package com.swervedrivespecialties.autonrefresh.util;
 
-import com.swervedrivespecialties.autonrefresh.RobotMap;
-
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj2.command.button.Button;
 
 public class BeakXboxController{
 
-    private XboxController controller;
+    //these constants all come from the layout of our physical controllers
+    //and their IO interfaces with WPILib
+
+    //deadband
+    private static final double DEADBAND = .05;
```

Comment:
I moved these here because it doesn't make much sense to have then in RobotMap. In general, things from util should not point to things outside the folder, for year-over-year reasons. These are still tunable, but the class is clearly BeakSquad Specific, so that's fine. If you really wanted to make them a year-by-year-thing, the best way to do it would be to create a subclass with all of these constants called BeakXboxConstants and then pass that in as an arg to the controller, then define an instance of that class in RobotMap and pass that in to all controllers, preferably with a default constructor that uses the current constants. I don't think that's worthwhile, though